README.md

```diff
@@ -93,6 +93,7 @@ components:
 
 |Name|Description|Required|Default|
 |---|---|---|---|
+|generateOppositeSide|Generate an application that will publish messages to `publish` operation of channels and read messages from `subscribe` operation of channels. Literally this flag will simply swap `publish` and `subscribe` operations in the channels. Be aware, generation could be incomplete e.g. if bindings are not defined for the opposite side.|No|`false`|
```

Comment:
make is clearer what is the use case and not only what the flag is changing, so the user knows what to use it for, that you are thinking about app client generation here but that generating main app and client from the same asyncapi file might be possible but really is a super simple scenario, otherwise some manual changes after generation will be needed right?

Reply:
of course, this template does not provide production ready application and changes required even for main generation case 😃 
description updated

Reply:
still, you provided a good explanation for the flag, use case to generate a mock that you could use for testing is pretty good